docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1960 @@
+{
```

Comment:
<p>What's up with this warning?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I have no idea, I merged `main` into my developer branch.

Reply:
maybe https://askubuntu.com/questions/1406952/what-is-the-meaning-of-this-pkgresourcesdeprecationwarning-warning-from-pipenv?

Reply:
<p>Seems to be gone! Is this something we need to fix in PyMC or was specific to your installation? (I did not read the links yet)</p><p> </p>

Reply:
<p>It is gone because re-ran the cell twice. I believe it has to do with my installation as in the the PR preview is not there https://pymcio--5721.org.readthedocs.build/projects/docs/en/5721/learn/core_notebooks/pymc_aesara.html#introduction-to-aesara</p>